.github/workflows/move-ready-for-review-prs.yml

```diff
@@ -64,5 +62,5 @@ jobs:
               position: 'bottom',
               column_id: initial_review.id,
             }).catch(error => {
-              console.error("Error occured while moving card to 'Initial Review'!")
+              console.error("Error occured while moving card to 'Initial Review': {error}")

```

Comment:
This won't do anything; you need to also switch the kind of quotes (to `` ` `` strings)

Reply:
Crap I missed that

Reply:
Wait I did change that to backticks, I believe you are seeing outdated diff somehow...

Reply:
huh, odd.